youtube_dl/extractor/lcp.py

```diff
@@ -0,0 +1,137 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none
+)
+
+
+class LcpIE(InfoExtractor):
+    IE_NAME = 'LCP'
+    _VALID_URL = r'https?:\/\/(?:www\.)?lcp\.fr\/(?:[^\/]+/)*(?P<id>[^/]+)'
+
+    _TESTS = [{
+        'url': 'http://www.lcp.fr/la-politique-en-video/schwartzenberg-prg-preconise-francois-hollande-de-participer-une-primaire',
+        'md5': 'aecf5a330cfc1061445a9af5b2df392d',
+        'info_dict': {
+            'id': 'd56d03e9',
+            'url': 're:http://httpod.scdn.arkena.com/11970/d56d03e9_[0-9]+.mp4',
+            'ext': 'mp4',
+            'title': 'Schwartzenberg (PRG) préconise à François Hollande de participer à une primaire à gauche'
+        }
+    }, {
+        'url': 'http://www.lcp.fr/emissions/politique-matin/271085-politique-matin',
+        'md5': '6cea4f7d13810464ef8485a924fc3333',
+        'info_dict': {
+            'id': '327336',
+            'url': 're:http://httpod.scdn.arkena.com/11970/327336_[0-9]+.mp4',
+            'ext': 'mp4',
+            'title': 'Politique Matin - Politique matin'
+        }
+    }]
+
+    def _real_extract(self, url):
+        """Extracts the information for a given url and returns it in a dictionary"""
+        display_id = self._match_id(url)
+
+        # Extract the web page
+        webpage = self._download_webpage(url, display_id)
+
+        # Extract the required info of the media files
+        media_files_info = self.__extract_from_webpage(display_id, webpage)
+        # Some web pages embed videos from other platforms like dailymotion, therefore we pass on these URL
+        if media_files_info is None:
+            return self.url_result(self.__extract_embed_url(webpage))
```

Comment:
If you are not detecting embeds yourself (you does not) you should just delegate to generic extractor `self.url_result(url, 'Generic')`.